scripts/initial_setup.sh

```diff
@@ -16,8 +16,8 @@ fi
 # Upload new challenge file to the challenges folder.
 echo "put challenge" | $connect_to_sftp_server:challenges
 
-MESSAGE="The ceremony is ready to get started! Let's do it. The first challenge was uploaded here:$SFTP_ADDRESS:challenges"
-. /app/scripts/e2e-test_no_new_upload.sh $MESSAGE
+MESSAGE="The ceremony is ready to get started! The first challenge was uploaded here: ${SFTP_ADDRESS}:challenges"
+. /app/scripts/send_msg_to_gitter.sh "$MESSAGE"
```

Comment:
You totally read my mind with this fix! I was actually going to make the same ${SUGGESTION}